### ISSUE TITLE
ClassNotFoundException, standalone on 1.17

### ISSUE THREAD

**Issue by SlimeDog** [Minecrell/ServerListPlus#358]
Spigot 1.17-3119
ServerListPlusServer 3.5.0-b495

ServerListPlus.yml sans comments
```
--- !Status
Default:
  Description:
  - |-
    &r The server is temporarily down for maintenance
    &r Please check back (refresh) in a few minutes ...
  Players:
    Max:
    - 25
    Hover:
    - |-
      &awebsite: &rblackdog.straight8.com
      &awhitelist: &renabled - members only
      &aviaversion: &renabled
  Version:
    Protocol: 755
  Favicon:
    Files:
    - server-icon.png
--- !Plugin
Stats: false
PlayerTracking:
  Enabled: false
  Storage: !JSONStorage
    Enabled: true
    SaveDelay: 5m
Samples:
  Multiple: true
  DynamicPlayers: false
Unknown:
  PlayerName: player
  PlayerCount: ???
  Date: ???
  BanReason: some reason
  BanOperator: somebody
  BanExpirationDate: never
Favicon:
  Timeout: 10s
  RecursiveFolderSearch: false
  ResizeStrategy: SCALE
--- !Server
Address: :25565
Login:
  Message:
  - |-
    &a Hello %player%!
    &a The server is temporarily down for maintenance
    &e Please check back (refresh) in a few minutes ...
```
On server start-up and player connection attempt, latest.log:
```
[23:00:12] [main/DEBUG] [ByteBufUtil]: -Dio.netty.threadLocalDirectBufferSize: 0
[23:00:12] [main/DEBUG] [ByteBufUtil]: -Dio.netty.maxThreadLocalCharBufferSize: 16384
[23:00:12] [main/INFO] [NetworkManager]: Listening on [id: 0xe983610a, L:/[0:0:0:0:0:0:0:0%0]:25565]
[23:07:29] [epollEventLoopGroup-3-1/DEBUG] [Recycler]: -Dio.netty.recycler.maxCapacityPerThread: 4096
[23:07:29] [epollEventLoopGroup-3-1/DEBUG] [Recycler]: -Dio.netty.recycler.maxSharedCapacityFactor: 2
[23:07:29] [epollEventLoopGroup-3-1/DEBUG] [Recycler]: -Dio.netty.recycler.linkCapacity: 16
[23:07:29] [epollEventLoopGroup-3-1/DEBUG] [Recycler]: -Dio.netty.recycler.ratio: 8
[23:07:29] [epollEventLoopGroup-3-1/DEBUG] [Recycler]: -Dio.netty.recycler.delayedQueue.ratio: 8
[23:07:29] [epollEventLoopGroup-3-1/DEBUG] [AbstractByteBuf]: -Dio.netty.buffer.checkAccessible: true
[23:07:29] [epollEventLoopGroup-3-1/DEBUG] [AbstractByteBuf]: -Dio.netty.buffer.checkBounds: true
[23:07:29] [epollEventLoopGroup-3-1/DEBUG] [ResourceLeakDetectorFactory]: Loaded default ResourceLeakDetector: io.netty.util.ResourceLeakDetector@32f205b3
[23:07:29] [epollEventLoopGroup-3-1/INFO] [LoggerHandler]: Client connected [id: 0x9e5b032c, L:/216.218.245.194:25565 - R:/108.226.240.191:49885]
[23:07:29] [epollEventLoopGroup-3-1/WARN] [DefaultChannelPipeline]: An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
java.lang.NoClassDefFoundError: com/google/common/util/concurrent/internal/InternalFutureFailureAccess
	at java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:1010) ~[?:?]
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150) ~[?:?]
	at jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:855) ~[?:?]
	at jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:753) ~[?:?]
	at jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:676) ~[?:?]
	at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:634) ~[?:?]
	at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:182) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:519) ~[?:?]
	at java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:1010) ~[?:?]
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150) ~[?:?]
	at jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:855) ~[?:?]
	at jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:753) ~[?:?]
	at jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:676) ~[?:?]
	at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:634) ~[?:?]
	at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:182) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:519) ~[?:?]
	at java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:1010) ~[?:?]
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150) ~[?:?]
	at jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:855) ~[?:?]
	at jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:753) ~[?:?]
	at jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:676) ~[?:?]
	at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:634) ~[?:?]
	at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:182) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:519) ~[?:?]
	at com.google.common.cache.LocalCache$LoadingValueReference.<init>(LocalCache.java:3472) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at com.google.common.cache.LocalCache$LoadingValueReference.<init>(LocalCache.java:3476) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2134) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2045) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at com.google.common.cache.LocalCache.get(LocalCache.java:3962) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:3985) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4946) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at com.google.common.cache.LocalCache$LocalLoadingCache.getUnchecked(LocalCache.java:4952) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at net.minecrell.serverlistplus.core.favicon.FaviconCache.get(FaviconCache.java:64) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at net.minecrell.serverlistplus.server.ServerListPlusServer.handle(ServerListPlusServer.java:227) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at net.minecrell.serverlistplus.server.ServerListPlusServer.post(ServerListPlusServer.java:160) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at net.minecrell.serverlistplus.server.network.ClientHandler.handle(ClientHandler.java:74) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at net.minecrell.serverlistplus.server.network.protocol.packet.PacketStatusRequest.handle(PacketStatusRequest.java:33) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at net.minecrell.serverlistplus.server.network.ClientHandler.channelRead(ClientHandler.java:92) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:324) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:296) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:324) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:296) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:795) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:480) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:378) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[ServerListPlusServer-3.5.0-b495.jar:?]
	at java.lang.Thread.run(Thread.java:831) [?:?]
Caused by: java.lang.ClassNotFoundException: com.google.common.util.concurrent.internal.InternalFutureFailureAccess
	at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:636) ~[?:?]
	at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:182) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:519) ~[?:?]
	... 69 more
```

**Comment by Minecrell** [comment]
How is ServerListPlusServer related to Spigot? I'm confused :D

**Comment by Minecrell** [comment]
Ah, can reproduce this. Will investigate. Thanks for the report!

**Comment by SlimeDog** [comment]
Confirm fixed in 3.5.0-b498. Thanks.
